host/frontend/host-orchestrator/jsonws.go

```diff
@@ -15,44 +15,86 @@
 package main
 
 import (
+	"encoding/json"
 	"log"
+	"net"
 	"net/http"
 	"sync"
 
 	"github.com/gorilla/websocket"
 )
 
-// A websocket connection that can send and receive json objects.
+// A websocket connection that can send and receive JSON objects.
 // Only one thread should call Recv() at a time, Send() and Close() are thread safe
-type JsonWs struct {
+type JSONWs struct {
```

Comment:
Should this be [MixedCaps](https://golang.org/doc/effective_go#mixed-caps) style? `JsonWs` already followed the abbreviation rules mentioned below.

Because the Effective Go page is not specific on abbreviations, referencing the [c++ style guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules):

> For the purposes of the naming rules below, a "word" is anything that you would write in English without internal spaces. This includes abbreviations, such as acronyms and initialisms. For names written in mixed case (also sometimes referred to as "[camel case](https://en.wikipedia.org/wiki/Camel_case)" or "[Pascal case](https://en.wiktionary.org/wiki/Pascal_case)"), in which the first letter of each word is capitalized, prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC().

Reply:
Initialism in Go are supposed to be kept with all caps: JSON, HTTP, URL, etc.